### ISSUE TITLE
Where did those configurations come from?

### ISSUE THREAD

**Issue by 153957** [HiSPARC/publicdb#130]
Station 501 has some configurations in the public database for 2009/2010. I could not find their sources in the datastore. Were they entered manually or lost in data migrations? Or mislabeled somehow?

**Comment by davidfokkema** [comment]
Strange! Are all configurations from that period missing?

**Comment by 153957** [comment]
I did not check all, but 2009/12/31 and 2010/1/22 do not have a config table. There are configs from more dates between/around those dates. I also checked 2011/4/4 and it did have its config. So something went wrong for the old data..

**Comment by davidfokkema** [comment]
Are the time stamps close to midnight?

**Comment by 153957** [comment]
Nope, 8 am in the morning, I also checked the day before and after, also did not have a config table.
Also the blobs did not contain more entries than required for the number of events (times number of detectors).
